ui/src/views/OpenVPNRW.vue

```diff
@@ -1726,6 +1748,9 @@ export default {
             console.error(e);
           }
           context.accounts = success.accounts;
+          for (var i in context.accounts) {
+            context.accounts[i].Connectivity = (context.accounts[i].statistics !== null ) ? context.accounts[i].statistics.red_interface : "0.0.0.0";
```

Comment:
I think here `0.0.0.0` is wrong, because that IP means "anyone"
If the information is not available, can we just return `-` without breaking the sort algorithm?ing

Reply:
return a `-`  breaks the function and `0.0.0.0` is never displayed, just a trick cc @andre8244

Reply:
If `0.0.0.0` is never displayed, it's ok for me.